package.json

```diff
@@ -87,6 +87,7 @@
     "date-fns": "^2.1.0",
     "graphql": "^14.5.8",
     "graphql-relay": "^0.6.0",
+    "graphql-subscriptions": "^1.1.0",
```

Comment:
Needed to provide a default Pubsub instance in Shyft.

Reply:
got it